dataprocessing/data_utils.py

```diff
@@ -0,0 +1,75 @@
+import cv2
+import os
+import logging
+from pathlib import Path
+logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
+logger = logging.getLogger(__name__)
+
+
+def videos2frames_frame_number(inputdir: Path, outputdir: Path, frame_number: int):
+    """
+    convert video to frames based on the requested number of frames.
+    """
+    assert inputdir.is_dir()
```

Comment:
You could just throw an exception, that might be more appropriate than an assert. Or let the exception just get thrown when the variable is used incorrectly.

Reply:
fixed here aed17ab